codes/classical/properties/Quasi-Cyclic Low Density Parity Check (QC-LDPC)

```diff
@@ -0,0 +1,23 @@
+
+code_id: QC-LDPC
+
+name: 'Quasi-Cyclic LDPC Codes based on Pre-Lifted Protographs'
```

Comment:
Can this just be "Quasi-cyclic LDPC code"? Are there quasi-cyclic ldpc codes not based on protographs? @AlexanderBarg